apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/draft-post-modal/index.js

```diff
@@ -0,0 +1,35 @@
+import { Modal, Button, Flex, FlexItem } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import draftPostImage from './images/draft-post.svg';
+import './style.scss';
+
+const DraftPostModal = () => {
+	const { __ } = useI18n();
+
+	return (
+		<Modal open className="wpcom-block-editor-draft-post-modal">
+			<img src={ draftPostImage } alt={ __( 'Draft post', 'full-site-editing' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Draft posts.' )` *ES Score: 8*